library/Falcon/Handler/WPMail.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * WP Mail Handler
+ *
+ * @author Joe Hoyle <joe@huamanmade.co.uk>
+ * @package Falcon
+ * @subpackage Handlers
+ */
+class Falcon_Handler_WPMail implements Falcon_Handler {
+
+	public function __construct( $options ) {
+
+	}
+
+	public function check_inbox() {}
+
+	public function send_mail( $users, Falcon_Message $message ) {
+		$options = $message->get_options();
+
+		$from = Falcon::get_from_address();
+		$author = $message->get_author();
+
+		$messages = array();
+		foreach ( $users as $user ) {
+
+			$headers = array(
+				'From'         => sprintf( '%s <%s>', $author, $from ),
+				'Reply-To'     => sprintf( '%s', $message->get_reply_address( $user ) ),
+				'Content-Type' => 'text/html; charset=UTF-8',
+			);
+
+			// Set the message ID if we've got one
+			if ( ! empty( $options['message-id'] ) ) {
+				$headers['Message-ID'] = $options['message-id'];
+			}
+
+			// If this is a reply, set the headers as needed
+			if ( ! empty( $options['in-reply-to'] ) ) {
+				$original = $options['in-reply-to'];
+				if ( is_array( $original ) ) {
+					$original = isset( $options['in-reply-to'][ $user->ID ] ) ? $options['in-reply-to'][ $user->ID ] : null;
+				}
+
+				if ( ! empty( $original ) ) {
+					$headers['In-Reply-To'] = $original;
+				}
+			}
+
+			if ( ! empty( $options['references'] ) ) {
+				$references = implode( ' ', $options['references'] );
+				$headers['References'] = $references;
+			}
+
+			wp_mail(
```

Comment:
We should probably check the return value.

Reply:
Fixed